unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/ItemToggle.cs

```diff
@@ -84,8 +87,11 @@ public virtual void Initialize(WearableItem w, bool isSelected, int amount, NFTI
 
     public void OnPointerEnter(PointerEventData eventData)
     {
-        if (!ShowReplacementWarningPanel())
-            ShowHidingWarningPanel();
+        if(!ShowIncompatibleWarningPanel())
+        {
+            if (!ShowReplacementWarningPanel())
```

Comment:
This IF chain getting a bit hard to follow, I would refactor this just for clarity.

Something like
```C#
if (IsIncompatible())
{
    ShowIncompatibleWarning();
}
else if (IsGoingToReplaceSomething())
{
    ShowReplacementPanel();
}
else
{
    ShowHidingWarningPanel();
}
```
 
bool IsGoingToReplaceSomething